icons/discordjs/discordjs-plain.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128" version="1.1">
```

Comment:
Could you please provide a `plain-wordmark` version?

Reply:
How would I do that? (I’m new to SVG)

Reply:
Done! :D